AppCenter/AppCenter/Support/iOS.modulemap

```diff
@@ -4,6 +4,10 @@ framework module AppCenter {
   export *
   module * { export * }
   
+  explicit module Private {
+    header "MSACConstants+Flags.h"
```

Comment:
Why is it a separate module? Can we just use it like other headers?

Reply:
No, otherwise he is not works.